CONTRIBUTING.md

```diff
@@ -17,17 +17,30 @@ Contributions are always welcome! We only ask that you open an issue first so we
 ## Development setup
 * Clone this repo
 * Run `npm install` in '/vscode-edge-devtools'
-* On an administrator prompt execute the following commands (assuming your drive is located at C:\)
-  * `mkdir c:\edge\src\out\Release\gen\devtools`
-  * `mkdir c:\edge\src\third_party\devtools-frontend\src\front_end`
-* Download a copy of the Microsoft Edge (Chromium) build from [https://thirdpartysource.microsoft.com](https://thirdpartysource.microsoft.com), current extension version builds from version 81.0.416.
-* **Open** the zip file and (inside the zip file) navigate to:
-  * `[ZIP_FILE]:\src\third_party\devtools-frontend\src\front_end`
-  * copy the contents of the "front_end" folder and paste them into `c:\edge\src\third_party\devtools-frontend\src\front_end`
-* **Open** the zip file and (inside the zip file) navigate to:
-  * `[ZIP_FILE]:\src\out\Release\gen\devtools`
-  * copy the contents of the "devtools" folder and paste them into `c:\edge\src\out\Release\gen\devtools`
-* Set the `EDGE_CHROMIUM_PATH` environment variable to `c:\edge\src` (assuming your drive is located at C:\)
+* Download and Extract Edge source files

```

Comment:
@vidorteg , when you get the chance, could you try out these new contributing steps and give me any feedback? Thanks

Reply:
I like the idea of having this process automated, but I would prefer to use npm instead of a new bash, users already know javascript so let's make contribution even easier.

Reply:
Done.  PTAL